bin/provisioner.js

```diff
@@ -1 +1,95 @@
-//TODO: The same as server but without configuring api...
\ No newline at end of file
+'use strict';
+
+var debug = require('debug')('aws-provisioner:bin:provisioner');
+var Promise = require('promise');
+var base = require('taskcluster-base');
+var provision = require('../provisioner/provision');
+var aws = require('multi-region-promised-aws');
+var data = require('../provisioner/data');
+var AwsManager = require('../provisioner/aws-manager');
+var awsPricing = require('../provisioner/aws-pricing');
+var Cache = require('../lib/cache');
+var taskcluster = require('taskcluster-client');
+
+
+
+var launch = function(profile) {
+  var cfg = base.config({
+    defaults: require('../config/defaults.js'),
+    profile: require('../config/' + profile),
+    filename: 'taskcluster-aws-provisioner',
+    envs: [
+        'provisioner_publishMetaData',
+        'provisioner_awsInstancePubkey',
+        'provisioner_awsKeyPrefix',
+        'taskcluster_queueBaseUrl',
+        'taskcluster_authBaseUrl',
+        'taskcluster_credentials_clientId',
+        'taskcluster_credentials_accessToken',
+        'pulse_username',
+        'pulse_password',
+        'aws_accessKeyId',
+        'aws_secretAccessKey',
+        'azure_accountName',
+        'azure_accountKey',
+        'influx_connectionString'
+    ]
+  });
+
+  var allowedRegions = cfg.get('provisioner:allowedRegions').split(',');
+  var keyPrefix = cfg.get('provisioner:awsKeyPrefix');
+  var pubKey = cfg.get('provisioner:awsInstancePubkey');
+  var provisionerId = cfg.get('provisioner:id');
+
+  var influx = new base.stats.Influx({
+    connectionString:   cfg.get('influx:connectionString'),
+    maxDelay:           cfg.get('influx:maxDelay'),
+    maxPendingPoints:   cfg.get('influx:maxPendingPoints')
+  });
+
+  var WorkerType = data.WorkerType.setup({
+    table: cfg.get('provisioner:workerTypeTableName'),
+    credentials: cfg.get('azure'),
```

Comment:
Yes, but it would be nice if the production azure table credentials only ever exists on auth.taskcluster.net.

We'll probably setup a storage account dedicated to the services that only stores small things.. I have one now called `taskclusterv1`. We could use secondary credentials and then revoke them later, but it just sucks to copy and send credentials around, and having to reset them later... Let's fix initially, it's quicker than emailing them around.